integration_tests/test_domestic_migration.py

```diff
@@ -1,23 +1,111 @@
+import os
+from pathlib import Path
+
 import numpy as np
 import pandas as pd
 import pytest
+import yaml
+from vivarium.framework.utilities import from_yearly
 
 from vivarium_census_prl_synth_pop.constants import data_values, metadata, paths
 
+from .conftest import (
+    FuzzyChecker,
+    from_yearly_multiplicative_drift,
+    multiplicative_drift_to_bounds_at_timestep,
+    multiplicative_drift_to_bounds_through_timestep,
+)
 
-def test_individuals_move(simulants_on_adjacent_timesteps):
-    for before, after in simulants_on_adjacent_timesteps:
+
+@pytest.fixture(scope="module")
+def target_migration_rates(sim):
+    assert (
+        metadata.UNITED_STATES_LOCATIONS == []
+    ), "Integration tests do not support subsets by US state"
+
+    with open(
+        Path(os.path.dirname(__file__)) / "v_and_v_inputs/domestic_migration.yaml"
+    ) as f:
+        targets = yaml.safe_load(f)
+
+    targets["individual_migration_rate_per_year"]["total"] = sum(
+        [v for v in targets["individual_migration_rate_per_year"].values()]
+    )
+
+    targets["household"] = from_yearly(
+        targets["household_migration_rate_per_year"],
+        pd.Timedelta(days=sim.configuration.time.step_size),
+    )
+
+    targets["individual"] = {}
+    for k in targets["individual_migration_rate_per_year"].keys():
+        targets["individual"][k] = from_yearly(
+            targets["individual_migration_rate_per_year"][k],
+            pd.Timedelta(days=sim.configuration.time.step_size),
+        )
+
+    targets["multiplicative_drift"] = {
+        "lower_bound": from_yearly_multiplicative_drift(
+            targets["multiplicative_drift_per_year"]["lower_bound"],
+            time_step_days=sim.configuration.time.step_size,
+        ),
+        "upper_bound": from_yearly_multiplicative_drift(
+            targets["multiplicative_drift_per_year"]["upper_bound"],
+            time_step_days=sim.configuration.time.step_size,
+        ),
+    }
+
+    return targets
+
+
+def test_individuals_move(
+    simulants_on_adjacent_timesteps, fuzzy_checker: FuzzyChecker, target_migration_rates
+):
+    all_time_individual_movers = []
+
+    for time_steps, (before, after) in enumerate(simulants_on_adjacent_timesteps):
         individual_movers = before["household_id"] != after["household_id"]
-        assert individual_movers.any()
+        fuzzy_checker.fuzzy_assert_proportion(
```

Comment:
Here's an example of a converted assert -- instead of .any(), we check the proportion using the target. Most of the diff in the test files is these conversions.